### PR TITLE
Remove the annoying shim files, use plugin google-closure-library-webpack-plugin support webpack watch mode.

### PR DESCRIPTION
**Pull request by funte**
### Resolves

_What Github issue does this resolve (please include link)?_

### Proposed Changes

_Describe what this Pull Request does_

### Reason for Changes

_Explain why these changes should be made_

### Test Coverage

Mocha test passed.

### CODE REVIEW

File: core/connection.js [hunk 1]
```diff
@@ -673,7 +673,7 @@ Blockly.Connection.prototype.onCheckChanged_ = function() {
 Blockly.Connection.prototype.setCheck = function(check) {
   if (check) {
     // Ensure that check is in an array.
-    if (!goog.isArray(check)) {
+    if (!goog.isArrayLike(check)) {
```
Comment (apple502j) [inline core/connection.js:676]: `Array.isArray`?

File: core/field_dropdown.js [hunk 2]
```diff
@@ -302,7 +302,7 @@ Blockly.FieldDropdown.prototype.trimOptions_ = function() {
   this.prefixField = null;
   this.suffixField = null;
   var options = this.menuGenerator_;
-  if (!goog.isArray(options)) {
+  if (!goog.isArrayLike(options)) {
```
Comment (apple502j) [inline core/field_dropdown.js:305]: `Array.isArray`?

File: core/connection.js (hunk 1 above)
Comment (funte) [reply core/connection.js:676]: In google-closure-library@20200830.0.0 the goog.isArray removed, i just found this [goog.isArrayLike](https://google.github.io/closure-library/api/goog.html#isArrayLike)

Comment (apple502j) [reply core/connection.js:676]: Why not use the actual `Array.isArray`?

Comment (funte) [reply core/connection.js:676]: `goog.isArrayLike` accept string, `Array.isArray` is better!!

File: core/block_svg.js [hunk 3]
```diff
@@ -959,7 +959,7 @@ Blockly.BlockSvg.prototype.setWarningText = function(text, opt_id) {
   }
 
   var changedState = false;
-  if (goog.isString(text)) {
+  if (typeof text === string) {
```
Comment (moonkop) [inline core/block_svg.js:962]:
```suggestion
  if (typeof text === 'string') {
```

**Review by moonkop — CHANGES_REQUESTED** [review]

**Pull request closed without merging**